kube-controllers/pkg/controllers/node/ipam.go

```diff
@@ -278,6 +286,9 @@ func (c *ipamController) acceptScheduleRequests(stopCh <-chan struct{}) {
 				// built in to the cleanup code.
 				log.WithError(err).Warn("error syncing IPAM data")
 				kick(c.syncChan)
+				lastSyncTimestampGauges.WithLabelValues("failure").SetToCurrentTime()
```

Comment:
I think we probably want to also catch the syncs that occur a few lines above, in the Periodic IPAM sync branch no?